src/gui/mzroll/tabledockwidget.h

```diff
@@ -20,320 +20,381 @@ class NumericTreeWidgetItem;
-    QPalette pal;    
-    void showSameGroup(QPair<int, int> sameMzRtGroupIndexHash);
-    void deletePeaks();
-    void addRow(PeakGroup *group, QTreeWidgetItem *root);
-    void heatmapBackground(QTreeWidgetItem *item);
-    PeakGroup *readGroupXML(QXmlStreamReader &xml, PeakGroup *parent);
-    void writeGroupXML(QXmlStreamWriter &stream, PeakGroup *g);
-    void readPeakXML(QXmlStreamReader &xml, PeakGroup *parent);
-    /**
-     * @brief- it will add samples used to group being generated while creating from mzroll file
-    * @detail This method will add all sample to group being
-    *created from mzroll file. It will read SamplesUsed attribute of a group
-    *and if it's value is "Used", then assign this mzSample to that group
-    */
-    void readSamplesXML(QXmlStreamReader &xml, PeakGroup *group, float mzrollVersion);
-
-    /**@brief-mark varible <mzrollv_0_1_5> true or false
-     *@details  this method marks varible <mzrollv_0_1_5> true if loaded mzroll
-     * file is of v0.1.5 or older otherwise false based on one attribute
-     * <SamplesUsed> which is introduced here.
-    */
-    void markv_0_1_5mzroll(QString fileName);
-
-    void setupFiltersDialog();
-
-    QList<PeakGroup> allgroups;
-
-    TrainDialog *traindialog;
-    ClusterDialog *clusterDialog;
-    QDialog *filtersDialog;
-    QMap<QString, QHistogramSlider *> sliders;
-    float rtWindow = 2;
-    tableType type;
-    tableViewType viewType;
-    peakTableSelectionType peakTableSelection;
-    QList<PeakGroup *> getCustomGroups(peakTableSelectionType peakSelection);
-    bool tableSelectionFlagUp;
-    bool tableSelectionFlagDown;
+  QPalette pal;
+  void addRow(PeakGroup *group, QTreeWidgetItem *root);
+  void heatmapBackground(QTreeWidgetItem *item);
+  PeakGroup *readGroupXML(QXmlStreamReader &xml, PeakGroup *parent);
+  void writeGroupXML(QXmlStreamWriter &stream, PeakGroup *g);
+  void readPeakXML(QXmlStreamReader &xml, PeakGroup *parent);
+
+  /**
+   * @brief- it will add samples used to group being generated while creating
+   * from mzroll file
+   * @detail This method will add all sample to group being
+   * created from mzroll file. It will read SamplesUsed attribute of a group
+   * and if it's value is "Used", then assign this mzSample to that group
+   */
+  void readSamplesXML(QXmlStreamReader &xml,
+                      PeakGroup *group,
+                      float mzrollVersion);
+
+  /**
+   * @brief-mark varible <mzrollv_0_1_5> true or false
+   * @details  this method marks varible <mzrollv_0_1_5> true if loaded mzroll
+   * file is of v0.1.5 or older otherwise false based on one attribute
+   * <SamplesUsed> which is introduced here.
+   */
+  void markv_0_1_5mzroll(QString fileName);
+
+  // TODO: investigate and remove this dialog if not being used
+  void setupFiltersDialog();
+
+  TrainDialog *traindialog;
+  ClusterDialog *clusterDialog;
+  QDialog *filtersDialog;
+  QMap<QString, QHistogramSlider *> sliders;
+  float rtWindow = 2;
```

Comment:
@saifulbkhan Not being used.

Reply:
`rtWindow` is really not used anywhere. Will remove.